alacritty_terminal/src/term/mod.rs

```diff
@@ -2118,10 +2117,23 @@ impl ansi::Handler for Term {
     }
 
     #[inline]
-    fn set_scrolling_region(&mut self, region: Range<Line>) {
-        trace!("Setting scrolling region: {:?}", region);
-        self.scroll_region.start = min(region.start, self.grid.num_lines());
-        self.scroll_region.end = min(region.end, self.grid.num_lines());
+    fn set_scrolling_region(&mut self, top: usize, bottom: usize) {
+        if top >= bottom {
+            debug!("Invalid scroll region escape: ({};{})", top, bottom);
```

Comment:
I’m not sure that escape is accurate here. It was fine ansi.rs, since we’re working with escapes. Probably dropping escapes will make sense.

Reply:
Yeah you're right, feel free to fix this yourself.